packages/analytics-client-common/test/session.test.ts

```diff
@@ -0,0 +1,25 @@
+import { isNewSession } from '../src/session';
+
+describe('session', () => {
+  const sessionTimeout: number = 30 * 60 * 1000;
+
+  test('should be a new session for the first event', () => {
+    const _isNewSession = isNewSession(sessionTimeout, undefined);
+
+    expect(_isNewSession).toBe(false);
+  });
```

Comment:
Kind of confusing here that if the first event should init a new session, then we should expect `isNewSession()` to be true instead of false.

Reply:
In the init method [here](https://github.com/amplitude/Amplitude-TypeScript/blob/main/packages/analytics-browser/src/browser-client.ts#L75), we will set the sessionId at the beginning, the first event will compare with the init session.